AntWars/ConfigurationPanel.cs

```diff
@@ -53,6 +53,9 @@ private void btn_player1ConfigNew_Click(object sender, EventArgs e)
         {
             configLoader.newPlayer1();
             configLoadedOrNewCreatedPlayer1();
+            calculateAntCostsByPlayer(configLoader.get().Player1);
+            label__player1CarryCost.Text = configLoader.get().Player1.carryCost.ToString();
+            label__player1ScoutCost.Text = configLoader.get().Player1.scoutCost.ToString();
```

Comment:
Falls es zu schlimm aussieht kann ich den Spaß auch in zwei funktionen(für beide player) auslagern...